packages/coinjoin/src/utils/coordinatorUtils.ts

```diff
@@ -31,3 +42,119 @@ export const getScriptPubKeyFromAddress = (
 
     throw new Error('Unsupported scriptType');
 };
+
+// check WabiSabi.scriptPubKey format by OP and return AllowedScriptType
+export const getScriptTypeFromScriptPubKey = (scriptPubKey: string) => {
```

Comment:
nitpick: I'd verbosely annotate this function with AllowedScriptTypes. Could be more future proof 

```suggestion
export const getScriptTypeFromScriptPubKey = (scriptPubKey: string): AllowedScriptTypes => {
```